backend/src/main/java/com/google/rolecall/services/SectionServices.java

```diff
@@ -343,7 +369,7 @@ private SiblingError updateSiblingPosition(Integer positionId,
               .setNotes(sibling.getNotes())
               .setOrder(sibling.getOrder())
               .setSiblingId(siblingId == -1 ? sibling.getSiblingId() : siblingId)
-              .setSize(sibling.getSiblingId())
+              .setSize(null)
```

Comment:
If it's possible for this to be null, then the frontend type needs to also be adjusted, right? Currently it looks like we have it defined as `size: number` which means it can never be null or undefined, and if it ever is then we could have bugs on the frontend.

Reply:
Good catch! I changed null to -1 which also matched what I did elsewhere.